setonix/configs_site_allusers/config.yaml

```diff
@@ -36,21 +36,21 @@ config:
   # identifies Spack staging to avoid accidentally wiping out non-Spack work.
   build_stage:
 #    - $tempdir/$user/spack-stage
-    - /software/$PAWSEY_PROJECT/$USER/setonix/software/build_stage
+    - /software/$PAWSEY_PROJECT/$USER/setonix/software/$user/build_stage
```

Comment:
Here and below: just for consistency, I think we should keep the same capitalisation for spack config variables. They are case insensitive, but environment variables are not, and having them different makes it look like they are not referring to the same thing.

Reply:
Thanks Sam, I am going to enforce consistency, for those entries that are entirely Pawsey-made.

I think in this context there is case sensitivity, but Spack provides a bunch of lowercase internal variables, such as `$spack_root`, `$tempdir`, and `$user`, which can then generate ambiguity/confusion around case insensitivity in those overlapping cases, such as USER.